src/android/config.js

```diff
@@ -1,92 +1,297 @@
-            .first()
-            .selector(record);
+    const referenceRange = selectFrom(standardOptions.series)
+      .where({ label: TARGET_NAME })
+      .first()
+      .selector(record);
 
-
-        return (
-            <div>
-                <div className={classes.title}>
-                    {new Date(record.pushDate).format('yyyy-MM-dd')}
-                </div>
-                <div>
+    return (
+      <div>
+        <div className={classes.title}>
+          {new Date(record.pushDate).format("yyyy-MM-dd")}
+        </div>
+        <div>
           <span
-              style={{ backgroundColor: series.style.color }}
-              className={classes.tooltipKey}
+            style={{ backgroundColor: series.style.color }}
+            className={classes.tooltipKey}
           />
-                    {series.label}
-                    {' '}:
-                    {round(record.result, { places: 3 })}
-                </div>
-                <div>
-                    {(() => {
-                        const diffMax = record.result - referenceRange.max;
-                        const diffMin = record.result - referenceRange.min;
+          {series.label} :{round(record.result, { places: 3 })}
+        </div>
+        <div>
+          {(() => {
+            const diffMax = record.result - referenceRange.max;
+            const diffMin = record.result - referenceRange.min;
 
-                        if (diffMax > 0) {
-                            return (
-                                <span className={classes.above}>
+            if (diffMax > 0) {
+              return (
+                <span className={classes.above}>
                   {`${round(diffMax, {
-                      places: 2,
+                    places: 2
                   })}ms above target`}
                 </span>
-                            );
-                        }
-                        if (diffMin < 0 ){
-                            return (
-                                <span className={classes.below}>
-                {`${round(-diffMin, {
-                    places: 2,
-                })}ms below target`}
-              </span>
-                            );
-                        }
+              );
+            }
+            if (diffMin < 0) {
+              return (
+                <span className={classes.below}>
+                  {`${round(-diffMin, {
+                    places: 2
+                  })}ms below target`}
+                </span>
+              );
+            }
 
-                        return (
-                            <span className={classes.between}>meets target</span>
-                        );
+            return <span className={classes.between}>meets target</span>;
+          })()}
+        </div>
+      </div>
+    );
+  }
+);
 
+const PLATFORMS = [
+  {
+    id: "g5",
+    label: "Moto G5 (arm7)",
+    filter: {
+      eq: {
+        platform: "android-hw-g5-7-0-arm7-api-16"
```

Comment:
I'm not sure if there's an `options` field available here, but if so, it should use `pgo`.

Reply:
I am not sure either but I added it anyway. Thanks for your review @gmierz :)